src/viam/sdk/tests/test_encoder.cpp

```diff
@@ -7,14 +7,8 @@
 
 #include <boost/test/included/unit_test.hpp>
 
-#include <viam/api/common/v1/common.pb.h>
```

Comment:
I stripped lots of direct `#include <viam/api` where I could. There are still some in our public headers. We should strive to eliminate them all.

Reply:
Just created a [ticket](https://viam.atlassian.net/browse/RSDK-6734) for this.